include/openssl/ec.h

```diff
@@ -1189,7 +1189,7 @@ int ECDSA_SIG_set0(ECDSA_SIG *sig, BIGNUM *r, BIGNUM *s);
  *  \return pointer to a ECDSA_SIG structure or NULL if an error occurred
  */
 ECDSA_SIG *ECDSA_do_sign(const unsigned char *dgst, int dgst_len,
-                         EC_KEY *eckey);
+                         EC_KEY *eckey, int type);
```

Comment:
Don't change a public function's prototype. If needed add a new public function. I suggest we only add support for it in the EVP API.